Makefile

```diff
@@ -0,0 +1,5 @@
+install:
+	pip install hatch
```

Comment:
installing Hatch with pip might not be the best idea

> This method modifies the Python environment in which you choose to install. Consider instead using [pipx](https://hatch.pypa.io/latest/install/#pipx) to avoid dependency conflicts.
https://hatch.pypa.io/latest/install/#pip